Add single argument version of `isapprox` (#32305)
JuliaLang/julia: NEWS.md

  otherwise ([#24353]).
* `IPAddr` subtypes now behave like scalars when used in broadcasting ([#32133]).
* `clamp` can now handle missing values ([#31066]).

> * `isapprox` (or `≈`) now has a one-argument "curried" method which returns a function, like `==(x)`.

Reviewer: ```suggestion
* `isapprox` (or `≈`) now has a one-argument "curried" method which returns a function, like `==(x)` ([#32305]).
```
Sorry, forgot this in my earlier suggestion for this line. NEWS entries include a reference to an issue or PR.

> * * `isapprox` (or `≈`) now has a one-argument bersion which creates a comparison function, like `==(x)`.

Reviewer: ```suggestion
* `isapprox` (or `≈`) now has a one-argument "curried" method which returns a function, like `==(x)`.
```

> * `isapprox` (or `≈`) now has a one-argument "curried" method which returns a function, like `==(x)` ([#32305]).

Reviewer 2: Maybe good to make the comparison with `==` even more explicit?

```suggestion
* `isapprox` (or `≈`) now has a one-argument "curried" method `isapprox(x)` which returns a function, like `isequal` (or `==`)` ([#32305]).
```
